include/zos-sys-info.h

```diff
@@ -31,8 +31,14 @@ typedef enum {
 
 // CPU Management Control Table (CCT).
 typedef struct ZOSCCT {
-  uint8_t filler[110];
-  uint16_t cpuCount; // Number of online CPUs.
+  uint8_t filler1[72]; // 0:72 Ignore fields not relevant to current implementation 
```

Comment:
can you add a link to the doc where this comes from?

Reply:
This part of the code was adapted from the OMR code here: https://github.com/eclipse/omr/blob/d810fcb9869df81172f2e7b87d7fe590304f397c/port/zos390/omrsimap.h#L68C4-L68C32 

After searching for the origin, it turns out it's not documented. Others have had issues finding the documentation: https://bit.listserv.ibm-main.narkive.com/ouvgcgK6/srm-cpu-mgmt-table-layout

But it is in SYS1.MODGEN(IRACCT)